ui/app/src/components/RecentActivityDashlet/RecentActivityDashlet.tsx

```diff
@@ -163,17 +163,14 @@ export default function RecentActivityDashlet() {
 
   // region Item Updates Propagation
   useEffect(() => {
-    const events = [itemsDeleted.type, itemsRejected.type, itemsApproved.type, itemsScheduled.type];
+    const events = [deleteContentEvent.type, workflowEvent.type, publishEvent.type]; // TODO: should contentEvent be here?
```

Comment:
Re "TODO": if content event ever changes the content of this widget, it should. Otherwise, no.